kuma/wiki/templates/wiki/document.html

```diff
@@ -367,6 +368,12 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
                       {{ datetimeformat(current_revision.created, format='datetime') }}
                     </div>
                   {% endif %}
+		 {% if is_admin %}
+		    <div class="contributors-sub">
+		      Document number: {{ document.id }}
```

Comment:
There's a how to guide for this here: http://kuma.readthedocs.org/en/latest/localization.html